packages/block-editor/src/components/global-styles/color-panel.js

```diff
@@ -228,15 +230,38 @@ function ColorPanelDropdown( {
 							{ 'is-open': isOpen }
 						),
 						'aria-expanded': isOpen,
+						ref: colorGradientDropdownButtonRef,
 					};
 
 					return (
-						<Button __next40pxDefaultSize { ...toggleProps }>
-							<LabeledColorIndicators
-								indicators={ indicators }
-								label={ label }
-							/>
-						</Button>
+						<>
+							<Button { ...toggleProps } __next40pxDefaultSize>
+								<LabeledColorIndicators
+									indicators={ indicators }
+									label={ label }
+								/>
+							</Button>
+							{ hasValue() && (
+								<Button
+									__next40pxDefaultSize
+									label={ __( 'Reset' ) }
+									aria-label={
+										sprintf(
+											/* translators: %s: The name of the color property being reset e.g. "Background" */
```

Comment:
While this is OK for now, something to note is that we'll likely be changing how we provide translator comments in the future - see https://github.com/WordPress/gutenberg/discussions/67275 for details.

Reply:
Thanks for the heads-up! I’ll definitely keep an eye on that.